canvas_modules/common-canvas/src/notification-panel/notification-panel.scss

```diff
@@ -59,10 +59,9 @@ $toolbar-button-height: 41px; // Allow one extra pixel for the toolbar border
 		color: $text-02;
 		padding-top: $spacing-02;
 	}
-	.notification-panel-close-icon {
-		position: absolute;
-		right: $spacing-05;
-		top: ($spacing-05 + $spacing-02);
+	.notification-panel-close-button {
+		bottom: $spacing-02;
+		left: ($spacing-06 + $spacing-12);
```

Comment:
Please use `left: calc(#{$spacing-06} + #{$spacing-12});`
`cursor: pointer;` can be removed.
<img width="321" alt="Screen Shot 2023-02-10 at 10 41 13 AM" src="https://user-images.githubusercontent.com/25124000/218172253-86f3370a-4b9f-4df1-8a32-da838077f8b4.png">